src/validations/member.js

```diff
@@ -23,8 +21,6 @@ const validateUpdate = (req, res, next) => {
         return value;
       }),
     phone: Joi.string()
-      .positive()
-      .integer()
       .min(10000000)
```

Comment:
when the field is a string, min() and max() refers to the text length. You can use the validation provided by @OscarLucero in slack channel

Reply:
sorry for this